005-RockPaperScissorsBoom/Student/Guides/challenges/BuildCICDPipelineWithGithubActions.md

```diff
@@ -0,0 +1,45 @@
+# Challenge 7 - Build a CI/CD workflow with GitHub Actions
+
+## Prerequisites
+
+1. [Challenge 4 - Run the app on Azure](./RunOnAzure.md) should be done successfully.
+
+## Introduction
+
+In a previous challenge we manually deployed the app on Azure. Now with this challenge you will be able to build an entire CI/CD workflow with GitHub actions.
+
+![GitHub Actions CI/CD](images/07-CICDwGithubActions-arch.png)
+
+## Challenges
+
+1. Create a Build workflow with GitHub actions to build your Docker images and push it to your Azure Container Registry (ACR). Your workflow should use `Continuous Integration` to build and push a new image with every relevant commit.
+2. Create a Release workflow with GitHub actions to deploy your images to the Azure Web App Service for Containers previously provisioned. Your workflow should implement `Continuous Delivery` approach to deploying the new image.
+3. Update one file on your `master` branch and commit this change, it should trigger automatically the Build and the Release definitions to deploy the new version of your app.
+4. Once deployed, test the app as an end-user, and play a game once deployed there.
+
+## Success criteria
```

Comment:
Capitalize 'Criteria' here... headers should cap each word, same applies to the other headers in this file.